app/views/core_induction_programmes/years/_nqt_plus_one_banner.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="govuk-notification-banner govuk-!-margin-top-2 govuk-!-margin-bottom-6" role="region" aria-labelledby="govuk-notification-banner-title">
+  <div class="govuk-notification-banner__header">
+    <h2 class="govuk-notification-banner__title" id="govuk-notification-banner__header">
+      Important
+    </h2>
+  </div>
+  <div class="govuk-notification-banner__content">
+    <p class="govuk-notification-banner__heading">
+      Do not try to do all the modules
+    </p>
+
+    <p>You can only use these resources until the end of the 2022 summer term, so we recommend you:</p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>focus on development areas you feel need a boost</li>
+      <li>dip in and out of the resources rather than following the prescribed week by week structure</li>
+    </ul>
+
+    <p>You should also ignore references to mentor meetings and associated practical exercises featured in the resources.</p>
+  </div>
+</div>
```

Comment:
I thought govuk components had a banner. Can we use them? https://dfe-digital.github.io/govuk-components/#notification-banner

Reply:
(might also require upgrading govuk components to current version)